### ISSUE TITLE
Accessibility: low contrast on light theme for bottom icons in iOS

### ISSUE THREAD

**Issue by CuriousMagpie** [HabitRPG/habitrpg-ios#1210]
**Describe the bug**
From user:
> I'm submitting this feedback that, on iOS, you can barely see the icons at the bottom of the screen in light mode. As a user with visual impairments that struggle with dark mode, I usually default to light mode, but this lack of contrast makes it impossible to tell which icon are available.

I looked on my iPhone--which is almost always in light mode--and the icons are pretty light.

**Comment by saraolson** [comment]
we can change the light mode icons to gray 200 #878190
